### ISSUE TITLE
Git LFS install fails on Google Colab even after successful installation using package cloud installer script.

### ISSUE THREAD

**Issue by SachitNayak** [git-lfs/git-lfs#3605]
Hi,
I was trying to use git lfs on google Colaboratory. 

I **successfully** installed the package using:
`curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
`

However when I try to initialise with `git lfs install` it fails with the following output:
**_git: 'lfs' is not a git command. See 'git --help'.The most similar command is log_**
<br/>
Is this **because a reboot is required after running the script.deb.sh ?** or is it something else.
I'm not sure. It would really be nice to have git lfs working on google colab.

Thank You.

**Comment by bk2204** [comment]
Hey, thanks for writing in.

The script doesn't actually install the packages; it only sets up the repositories. So after that point you'll want to do a `sudo apt-get install git-lfs` to make sure the package for Git LFS is installed. The script works this way so if we had multiple packages, you'd get to pick what's installed instead of having to install everything.

No reboot is required; you should be fine to simply install the package and continue.

**Comment by julien-c** [comment]
Someone should suggest to the Colab team that they pre-install git-lfs.

**Comment by craigcitro** [comment]
From the colab side: we're taking a look at adding git-lfs support by default.

FWIW, feel free to ask on our issue tracker (https://github.com/googlecolab/colabtools/issues) whenever you've got feature requests like this.

**Comment by craigcitro** [comment]
... and git-lfs should be set up by default.

**Comment by julien-c** [comment]
really awesome @craigcitro, thanks! 

<img width="1107" alt="Screenshot 2022-04-21 at 21 55 29" src="https://user-images.githubusercontent.com/326577/164543331-6c31edf9-fd85-4a7c-9bb7-06b999db1a1a.png">

my only reservation is `v2.3.4` is a bit ancient, any way we could get a slightly more recent upgrade in the future?

**Comment by julien-c** [comment]
and a bonus suggestion would be to have `lfs` already setup, i.e. having a `/etc/config` or `~/.git/config` that already references the lfs filters (which are usually setup only once per system using `git lfs install`). Thanks again!

**Comment by BaosenZ** [comment]
> 

git-lfs is pre-installed in Colab. But I git clone my project, the git-lfs file in the project sill cannot download. Why is that?
